### PR TITLE
[Intel MKL] Enable DepthwiseConv2D bfloat16 fusions

### PR DESCRIPTION
**Pull request by CuiYifeng**

### CODE REVIEW

**Review by penpornk — COMMENTED** [review]
I'm sorry for the delay! I have a question please.

File: tensorflow/core/grappler/optimizers/mkl_remapper_test.cc [hunk 1]
```diff
@@ -22,6 +22,19 @@ limitations under the License.
...
+#ifdef ENABLE_INTEL_MKL_BFLOAT16
+#define REGISTER_TEST_BFLOAT16(TEST) \
+  REGISTER_TEST(TEST, DT_BFLOAT16, BFloat16Input);
+
+#define REGISTER_TEST_ALL_TYPES(TEST) \
+  REGISTER_TEST_FLOAT32(TEST);        \
+  REGISTER_TEST_BFLOAT16(TEST);
+#else
+#define REGISTER_TEST_ALL_TYPES(TEST) REGISTER_TEST_FLOAT32(TEST);
+#endif  // ENABLE_INTEL_MKL_BFLOAT16
```
[the remark this answers is not in the document]
Comment (CuiYifeng) [reply tensorflow/core/grappler/optimizers/mkl_remapper_test.cc:36]: Thank you for your advice! We are discussing it. I think we'd better to factor this common macro in another PR.

[the remark this answers is not in the document]
Comment (penpornk) [reply tensorflow/core/grappler/optimizers/mkl_remapper_test.cc:36]: Okay. I'll get this one in then. The rest of the PR looks good to me.

**Review by penpornk — APPROVED** [review]

**Comment by gbaned** [comment]
@CuiYifeng  Can you please check @penpornk's comments and keep us posted. Thanks!

**Comment by CuiYifeng** [comment]
@gbaned @penpornk Yes, of course. I will let you know when we have a conclusion, thanks!

**Comment by CuiYifeng** [comment]
@penpornk @gbaned Can we merge this PR first and factor this common macro in another PR later?

**Review by penpornk — APPROVED** [review]
Yes, I had already approved the PR anyway. It was ready to merge since last week.
The Windows failure looks unrelated. I'll rerun the tests again then we can pull it in.

**Pull request merged**